conftest.py

```diff
@@ -127,3 +131,43 @@ def pytest_configure(config):
     # tests, and indeed if we do they fail.
     settings.STATICFILES_STORAGE = (
         'django.contrib.staticfiles.storage.StaticFilesStorage')
+
+class CatalogMocker:
+    """A CatalogMocker.
+    A instance of CatalogMocker is a (reusable) context manager.
+    """
+    def __init__(self):
+        self.patcher = mock.patch('ideascube.serveradmin.catalog.Catalog',
+                                  spec=True)
+        self.packages = []
+        self.mocked = None
+
+    def create_package(self, id, **kwargs):
+        pseudo_package = Package(id, kwargs)
+        self.packages.append(pseudo_package)
+
+    def list_installed(self, ids_list):
+        if '*' in ids_list:
+            return self.packages
+
+        return [p for p in self.packages if p.id in ids_list]
+
+    def __enter__(self):
+        if self.mocked is not None:
+            raise RunTimeError(
+                "CatalogMocker is not a reentrant context manager")
```

Comment:
In general, I google the error message.
But it's not opposite to a more explicit message. I will add it.